util/changes.go

```diff
@@ -0,0 +1,35 @@
+package util
```

Comment:
Not sure if this is the right location for this function. Glad to move it if necessary.

Reply:
This belongs better in reflect.go